platform-sdk/platform-apps/demos/StatsDemo/src/main/java/com/swirlds/demo/stats/StatsDemoState.java

```diff
@@ -62,14 +69,48 @@ private static class ClassVersion {
 
     private static final long CLASS_ID = 0xc550a1cd94e91ca3L;
 
+    @Override
+    public void init(
+            Platform platform,
+            PlatformState platformState,
+            InitTrigger trigger,
+            SoftwareVersion previousSoftwareVersion) {
+        this.selfId = platform.getSelfId();
+    }
+
+    private NodeId selfId;
+
     public StatsDemoState() {}
 
     private StatsDemoState(final StatsDemoState sourceState) {
         super(sourceState);
+        this.selfId = sourceState.selfId;
     }
 
+    private static final AtomicInteger y = new AtomicInteger(60);
+    private static final AtomicInteger x = new AtomicInteger(60);
+
     @Override
-    public void handleConsensusRound(final Round round, final PlatformState platformState) {}
+    public void handleConsensusRound(final Round round, final PlatformState platformState) {
+
+        if (selfId.id() == 0) {
+            if (x.getAndDecrement() <= 0) { // Don't start sleeping for a little while
+                if (x.get() == 0) {
+                    System.out.println("<<<<<<<<<<<<< engaging throttle >>>>>>>>>>>>>>>>");
+                }
+                if (y.getAndDecrement() >= 0) { // don't sleep forever
+                    try {
+                        SECONDS.sleep(1); // TODO
```

Comment:
Note to reviewers: I am doing this for local testing. There should be no changes to this file that are merged when this PR merges.